src/command_modules/azure-cli-sf/azure/cli/command_modules/sf/_params.py

```diff
@@ -5,81 +5,97 @@
 from azure.cli.core.sdk.util import ParametersContext
 from azure.cli.core.util import get_json_object
 
-# For some commands we take JSON strings as possible
-with ParametersContext(command="sf application create") as c:
-    c.register("parameters", ("--parameters",), type=get_json_object,
-               help="JSON encoded list of application parameters.")
 
-with ParametersContext(command="sf application create") as c:
-    c.register("metrics", ("--metrics",), type=get_json_object,
-               help="JSON encoded list of application metrics and their \
-               descriptions.")
+# Global parameters
+# Documentation here can be omitted from command specification
+with ParametersContext(command="sf") as c:
+    c.argument("timeout", options_list=("--timeout", "-t"),
+               type=int,
+               help=("The server timeout for performing the operation, "
```

Comment:
I'd rather leave it here. Our `_help.py` is only for the command groups. Everything else is in the functions definition. It made sense to put the help for the timeout parameter here since it is a global parameter.

Also, is there an example of how also write documentation for a parameter in `_help.py`? I could only find commands or groups.